### PR TITLE
speaker list operations generate updates

### PR DESCRIPTION
**Pull request by susanthomson**

### CODE REVIEW

**Review by BRHoward — APPROVED** [review]
:ok_hand:

File: server/tweetSearch.js [hunk 1]
```diff
@@ -324,12 +324,22 @@ module.exports = function(client, fs, speakerFile) {
     }
 
     function addSpeaker(name) {
+        tweetUpdates.push({
```
Comment (stephen-lt-sl) [inline server/tweetSearch.js:327]: This looks pretty good, but it seems like it'd make more sense (given the other update objects) to have the speaker operations come under one type with an object property listing whether they were added or removed; functionally it's the same either way though, so if you have any reason for having it this way then keep it pretty much

**Review by stephen-lt-sl — APPROVED** [review]
Apart from the comment, :100:

File: server/tweetSearch.js (hunk 1 above)
Comment (susanthomson) [reply server/tweetSearch.js:327]: I had no particular reason for doing it that way so I've changed it, let me know if it's what you had in mind.

**Review by stephen-lt-sl — APPROVED** [review]

**Pull request merged**